cli/file_watcher.rs

```diff
@@ -53,33 +53,40 @@ impl Stream for Debounce {
       inner.event_detected.store(false, Ordering::Relaxed);
       Poll::Ready(Some(()))
     } else {
-      let _ = inner.interval.poll_tick(cx);
-      Poll::Pending
+      match Pin::new(&mut inner.delay).poll(cx) {
```

Comment:
can you `Pin` the `Debounce.delay` instead?

Reply:
I tried but I didn't know how I could make that compile, so instead I use `poll_unpin` here. How about this?

Reply:
Even better